internal/cli/roles_permissions.go

```diff
@@ -70,22 +81,38 @@ func listRolePermissionsCmd(cli *cli) *cobra.Command {
 				inputs.ID = args[0]
 			}
 
-			var list *management.PermissionList
+			list, err := getWithPagination(
```

Comment:
**Nit:** This change it outside the purview of the PR. It's ok for now, but next time consider separating these changes.